third-party/gtest-1.7.0/fused-src/gtest/gtest-all.cc

```diff
@@ -8676,8 +8676,8 @@ void Notification::WaitForNotification() {
 }
 
 Mutex::Mutex()
-    : type_(kDynamic),
```

Comment:
It already has: https://github.com/google/googletest/commit/33307529412166cd7633eee9bf0b0aff21b5cf52